oncotator/utils/FieldMapCreator.py

```diff
@@ -0,0 +1,111 @@
+from oncotator.utils.ConfigUtils import ConfigUtils
+
+
+class FieldMapCreator(object):
+    """Generic class for creating a mapping of field name to an annotation name.  Under the hood this is a simple dict.
+    """
+
+    @staticmethod
+    def choose_best_annotation(header, m, alternative_dict, deprioritize_input_annotations):
+        """  Choose the best annotation in mut for the given header.
+        :param header:
+        :param m:
+        :param alternative_dict:
```

Comment:
This comment should probably either fill in explanations for the params or remove the list of them.  Unless pydoc works differently than javadoc and needs the empty params.